utils.py

```diff
@@ -0,0 +1,18 @@
+def check_module(nwbfile, name, description=None):
```

Comment:
this function should already exist somewhere

Reply:
`from nwb_conversion_tools.utils.conversion_tools import get_module`